src/test/java/net/snowflake/client/jdbc/ResultSet0IT.java

```diff
@@ -17,7 +17,7 @@
 
 /** Result set test base class. */
 @Category(TestCategoryResultSet.class)
-public class ResultSet0IT extends BaseJDBCTest {
+public class ResultSet0IT extends BaseJDBCIT {
```

Comment:
we swithed the base class but in methods below it looks like we still create and use new connections - could you verify if we can reuse the connection here?

Reply:
I was able to change the set up methods to use the common-connection and I got rid of one of the init() methods that's no longer being used anywhere. I might be able to remove the other init() methods once I make the changes in the other PR's that have classes inheriting from ResultSet0IT